index.js

```diff
@@ -106,6 +80,8 @@ function FastForEach(spec) {
   this.as = spec.as;
   this.noContext = spec.noContext;
   this.noIndex = spec.noIndex;
+  // allow users to turn off the 'reuse/move DOM nodes' feature
+  this.allowMoveNodes = spec.allowMoveNodes === undefined ? true : spec.allowMoveNodes;
```

Comment:
haha. I don't like it. ;)  Well, more specifically, I think if someone wanted to turn it off they could override `shouldDelayDeletion`.